torch/linalg/__init__.py

```diff
@@ -82,6 +82,63 @@
     True
 """)
 
+inv = _add_docstr(_linalg.linalg_inv, r"""
+linalg.inv(input, *, out=None) -> Tensor
+
+This function computes the inverse matrix of a square matrix, or batch of such matrices, :attr:`input`.
```

Comment:
"inverse" is an overloading word that we should be careful about using alone like this.

Maybe "multiplicative inverse", in quotes, similar to how this is styled in the NumPy documentation? It's great that the next line defines what's meant.